modules/mini-games/Race.lua

```diff
@@ -504,25 +491,28 @@ Gui.element{
   width = 25
 }
 
-local maingui = 
+local maingui =
 Gui.element(function(_,parent)
     local main_flow = parent.add{ type = 'flow', name = "Race_flow"}
+
     dorpdown(main_flow)
+
     text_field_for_laps(main_flow)
+
     text_field_for_players(main_flow)
-    
 
 end)
 local function gui_callback(parent)
     local args = {}
     local flow = parent["Race_flow"]
 
-    local dorpdown = flow[dorpdown.name]
-    local fuel = dorpdown.get_item(dorpdown.selected_index)
+    local dorpdown_ = flow[dorpdown.name]
+
+    local fuel = dorpdown_.get_item(dorpdown_.selected_index)
     args[1] = fuel
 
-    local laps = flow[text_field_for_laps.name].text
-    args[2] = laps
+    local laps_ = flow[text_field_for_laps.name].text
+    args[2] = laps_
```

Comment:
Not a blocking change, but people tend to put the _ first, not after. Eg. `_laps`. Although I would suggest using `required_laps` since this will become the number of laps that are required.